external.config.js

```diff
@@ -1,7 +1,12 @@
 var path = require('path');
-var webpack = require('webpack');
+var UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 
 module.exports = {
+  /* webpack 4
```

Comment:
I believe it should be "webpack 3" - assuming this was left over

Reply:
There is some preliminary work to use webpack 4 that is commented out.  I can remove it, but wanted to preserve it in some commit for when we move to webpack 4.  Unfortunately, karma-webpack is not webpack 4 compatible *yet*.

Reply:
that is fine. I was just making sure.